test/extended/openstack/loadbalancers.go

```diff
@@ -382,6 +383,152 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 			e2e.Logf("Pod %s successfully accessed through svc loadbalancer on ip %s and port %d", podName, fip.FloatingIP, svcPort)
 		})
 	}
+
+	// Only valid with the Octavia amphora provider. The OVN-Octavia provider does not
+	// support allowed_cidrs in the LB (https://issues.redhat.com/browse/OCPBUGS-2789)
+	g.It("should limit service access on an UDP Amphora LoadBalancer when an UDP LoadBalancer svc setting the loadBalancerSourceRanges spec is created on Openshift", func() {
+
+		// Skipped for OVN-Octavia provider
+		skipIfNotLbProvider("Amphora", cloudProviderConfig)
+
+		g.By("Creating Openshift deployment")
+		depName := "udp-lb-sourceranges-dep"
+		svcName := "udp-lb-sourceranges-svc"
```

Comment:
We only use `svcName` once, I'd probably put it directly in the `NewTestJig()` call.

Reply:
Done